vagrant/boxes.d/00-debian.yaml

```diff
@@ -1,5 +1,5 @@
-debian10:
-  box_name: 'debian/buster64'
+debian11:
+  box_name: 'debian/bullseye64'
   # box_check_update: false
   image_name: !ruby/regexp '/Debian 10.*/'
```

Comment:
I guess that needs changing too.
If not, what is it even for?

Reply:
I think the original plan was that the sandbox and source environments would reference this definition.

Instead, I think these are just simple ways for us to test out the OS before we run our ansible roles against them.

Reply:
Oh, i'm talking about:
`image_name: !ruby/regexp '/Debian 10.*/'`